react/index.js

```diff
@@ -1,15 +1,15 @@
-let {
+import {
```

Comment:
Why do you replace `require` to `import`?

Reply:
Sorry, my bad. Patched in 20cd173e05fa57201ebb39bda6f7297e36426962